src/Credentials.js

```diff
@@ -197,20 +203,20 @@ class Credentials {
    *  @example
    *  credentials.createVerification({
    *   sub: '5A8bRWU3F7j3REx3vkJ...', // uPort address of user, likely a MNID
-   *   exp: <future timestamp>,
    *   claim: { name: 'John Smith' }
-   *  }).then( credential => {
+   *  }, 14*24*60*60 ).then( credential => {
    *   ...
    *  })
    *
    * @param    {Object}            [credential]           a unsigned claim object
    * @param    {String}            credential.sub         subject of credential (a valid DID)
    * @param    {String}            credential.claim       claim about subject single key value or key mapping to object with multiple values (ie { address: {street: ..., zip: ..., country: ...}})
-   * @param    {String}            credential.exp         time at which this claim expires and is no longer valid (seconds since epoch)
+   * @param    {number}            credential.exp         Optional time at which this claim expires and is no longer valid (seconds since epoch)
+   * @param    {number}            expiresIn              Optional validity time in seconds. Use either this or credential.exp
    * @return   {Promise<Object, Error>}                   a promise which resolves with a credential (JWT) or rejects with an error
    */
-  createVerification({ sub, claim, exp, vc }) {
-    return this.signJWT({ sub, claim, exp, vc })
+  createVerification({ sub, claim, exp, vc, callbackUrl }, expiresIn) {
+    return this.signJWT({ sub, claim, exp, vc, callbackUrl }, expiresIn)
```

Comment:
Having the `signJWT` method handle an `expiresIn` param seems like mixing responsibilities to me. It should only be concerned with signing the payload passed to it without having to also handle potentially deriving the `exp` timestamp. I suggest removing `exp` from the first argument of `createVerification` and changing the second argument to be an options object that can have either `expiresIn` or `exp`.  Then it will be the job of the `createVerification` to calculate `exp` if the options object contains `expiresIn`, or just pass it directly through to `signJWT`. It will also be cleaner to validate that no more than one of the options was passed in without having to add more unnecessary logic to `signJWT`.

Reply:
This makes sense to me -- since it requires changes in did-jwt, let's revert this for now and address it in a subsequent PR.